assets/workflow/Islet Physiology Studies/Islet physiology protocol 1.html

```diff
@@ -1,94 +1,171 @@
-<br />
-
-  <ol type="1">
-  <li>Start  perfusion with a 30 min washout period without substrate (Buffer A)</li>
-  <li>Switch  to 4mM AAM buffer B for 30 min in the absence of glucose to stimulate glucagon  secretion</li>
-  <li>Switch  to Buffer C with 4 mM AAM plus 3 mM glucose for 20 min</li>
-  <li>Switch  to Buffer D with 4 mM AAM plus 16.7 mM glucose for 20 min</li>
-  <li>Switch  to Buffer E with 4 mM AAM plus 16.7 mM glucose plus 0.1 mM IBMX</li>
-  <li>Switch  to Buffer A with no substrate for 20 min</li>
-  <li>After the initial 20 min of washout, add  30 mM KCl to Buffer A period for an additional 20 mins</li>
-  </ol>
-  <p></p>
-  <font size="+1"><li><strong>Insulin and Glucagon measurement</strong></li></font>
-  <br />
-  <ol type="1">
-  <li>Insulin and glucagon concentrations in the outflow samples were measured by the Radioimmunoassay and Biomarkers Core using human insulin RIA HI-14k, EMD Millipore and glucagon RIA GL-32k, EMD Millipore.</li>
-  </ol>
-  <br/>
-  <hr />
-  <font size="+1" align="left"><b>References</b></font>  
-  </li>
- 
-</ol>
-<ul>
-<strong><sup>1</sup></strong>Amino Acid  Mixture (AAM) in mM: 0.44 alanine, 0.19 arginine, 0.038 aspartate, 0.094  citrulline, 0.12 glutamate, 0.30 glycine, 0.077 histidine, 0.094 isoleucine,  0.16 leucine, 0.37 lysine, 0.05 methionine, 0.70 ornithine, 0.08 phenylalanine,  0.35 proline, 0.57 serine, 0.27 threonine, 0.073 tryptophan, and 0.20 valine, 2  mM glutamine.
-</ul>
-</body>
-</html>
+<html>
+  <head>
+    <title>Perifusion-UPenn-</title>
+    <style>
+      img {
+        display: block;
+        margin-left: auto;
+        margin-right: auto;
+      }
+    </style>
+  </head>
+
+  <body>
+    <img
+      src="/assets/workflow/Islet Physiology Studies/HPAP-logo.jpg"
+      alt="perifusion"
+      width="200"
+      height="90"
+      style="vertical-align: middle"
+    />
+    <strong
+      ><h3 align="center">
+        PERIFUSION TO MEASURE INSULIN AND GLUCAGON SECRETION RATE
+      </h3></strong
+    >
+
+    <p align="center">
+      Doris Stoffers Lab&#44; University of Pennsylvania<br />
+      Date revised&#58; August 10&#44; 2020
+    </p>
+    <ol type="I">
+      <font size="+1">
+        <li><strong>Equipments used for experiment</strong></li>
+      </font>
+      <br />
+
+      <ol type="1">
+        <li>
+          &nbsp;A computer&#45;controlled&#44; fast&#45;performance LPC system
+          &#40;BioRad Econo pump system with gradient former and buffer
+          selector&#41; that allowed programmable rates of flow and glucose
+          concentration in the perfusate
+        </li>
+        <li>A water bath &#40;37 &#8451;&#41;</li>
+        <li>A fraction collector &#40;Waters Division of Millipore&#41;</li>
+      </ol>
+      <br />
+
+      <font size="+1">
+        <li><strong>Set&#45;up</strong></li>
+      </font>
+      <br />
+
+      <ol type="1">
+        <li>
+          Prepare Krebs buffer &#40;pH 7&@46;4&#41; containing 114 mmol&#47;l
```

Comment:
Is it pH 7.4? I think period '.' is safe in html we can keep it for readability. If you want to convert to encoded character, it should be &#46.
<img width="1232" alt="image" src="https://github.com/faryabiLab/hpap-public/assets/56408423/56ec9148-df00-4c3b-8e49-ce6e3e0f89e5">